docs/content/ecs/authority.md

```diff
@@ -10,6 +10,16 @@
 
 ## How authority is represented in Unity's ECS
 
+### Authority shared components
+
+For every [SpatialOS component]({{urlRoot}}/content/glossary#spatialos-component) attached to a SpatialOS entity, the SpatialOS GDK for Unity (GDK) attaches a _ECS shared component_ (an ECS `ISharedComponentData`) to the ECS entity. This component is called `ComponentName.ComponentAuthority`, where `ComponentName` is the name of the SpatialOS component.
+
+This component contains a single field, `HasAuthority`, a `bool` that indicates whether the worker instance has authority over the SpatialOS component.
+
+> Note that this component does not contain information about `AuthorityLossImminent` notifications. To listen to these, see [Authority reactive components](#authority-reactive-components) below.
```

Comment:
Do we say "listen"? Maybe "receive updates on" ? And is it the worker instance which is getting updates?

And do we mean:
```suggestion
> Note that this component does not contain information about `AuthorityLossImminent` notifications. For information on how a worker instance gets updates on these, see [Authority reactive components](#authority-reactive-components) below.
```

Reply:
I'd hesitate to use `updates` since its highly associated with component updates

Reply:
```> Note that this component does not contain information about `AuthorityLossImminent` notifications. For information on how a worker instance gets notified about these, see [Authority reactive components](#authority-reactive-components) below.```

?

Reply:
LGTM!!! 
Completely understand about updates.